### ISSUE TITLE
[BUG]: unable to open playlist 

### ISSUE THREAD

**Issue by eberkopec** [yuliskov/SmartTube#4057]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

24.52

### Device Type

Smart TV/Box

### Affected Android

Fire OS 7.6.6.4

### Steps to reproduce the bug

Go to playlists
Click on a playlist
Buffering circle of death appears and playlist does not open

### Actual behavior

I tried opening a playlist that I created, and I just got the blue circle continuously circling in the playlist not opening.

### Additional information

Toshiba Amazon Fire TV

**Comment by EdmundGerber** [comment]
I'm seeing the same behaviour, on 24.52. I noticed that if I long press on a playlist, and chose 'add playlist to sidebar', I can access those videos just fine, from the sidebar. But I still cannot get the playlists to open normally.

**Comment by eberkopec** [comment]
Thank you. I will try that.Sent from my iPhoneOn Nov 2, 2024, at 4:58 PM, EdmundGerber ***@***.***> wrote:﻿
I'm seeing the same behaviour, on 24.52. I noticed that if I long press on a playlist, and chose 'add playlist to sidebar', I can access those videos just fine, from the sidebar. But I still cannot get the playlists to open normally.

**Comment by EdmundGerber** [comment]
This was never an issue until I updated to 24.52. I never noticed it mention private on any playlist before that version, wither, and not sure why it's happening now.

**Comment by serzhshakur** [comment]
Can this be related to the amount of videos in a playlist? I have 2 playlists with ~150 videos in each which I cannot open. However other playlists with only few videos open with no issues

**Comment by eberkopec** [comment]
I don’t think so. Most of my playlists have between 20 and 30, and some have as few as nine or 10. It was one of the smaller ones that I tried to open.Sent from my iPhoneOn Nov 3, 2024, at 3:45 PM, Sergey ***@***.***> wrote:﻿
Can this be related to the amount of videos in a playlist? I have 2 playlists with ~150 videos in each which I cannot open. However other playlists with only few videos open with no issues

**Comment by koffeedude** [comment]
I have the same issue with Fire TV Cube it just started happening after I updated to the latest version 24.52. The circle spins and spins and nothing works. I deleted cache, restarted and checked Fire TV updates and its still happening.

**Comment by 44cK3r** [comment]
I have the same problem but I have it set up so I can just click and hold and it will open just fine I don't remember how I did it but I think I just removed everything from that menu that would pop up except for open channel I can't remember that well so that may not be what I actually did sorry

**Comment by hehehe121213** [comment]
I found out what's wrong. Seems like now the app  couldn't access Private playlists. I using my phone changes my playlist to Unlisted and now the app on tv can access it again

**Comment by Salamat-UK** [comment]
Am not sure why #4054 was closed, referring to this issue? Have updated to the latest but still getting the Subscriptions issue! (On multiple devices)

**Comment by dontwantonebutneedone** [comment]
> Can this be related to the amount of videos in a playlist? I have 2 playlists with ~150 videos in each which I cannot open. However other playlists with only few videos open with no issues

No. I tried that last night. Any playlist other than the pre-setup Watch Later and Liked Videos won't open. I tried reducing the # of videos from 17 down to 1 and it still wouldn't play.

I noticed that they play on the regular youtube client, but  on my computer chrome browser with adblock+ extension, each of them sat there for 15+ seconds, popped an ad, asked me if I wanted to get youtube premium again, and then played the videos.

I also let it sit for about 2 hours to see if it'd eventually play (nope), dropped the resolution, changed the network engine and none of that helped.

**Comment by Salamat-UK** [comment]
> Am not sure why #4054 was closed, referring to this issue? Have updated to the latest but still getting the Subscriptions issue! (On multiple devices)

Subscriptions work on regular YT., ok. Was just looking through my playlists on Smart tube and I see that my 'Liked' videos are on exactly 5000! I can't do anything to open this list and delete some. Is this purely a coincidence that it's exactly '5000'? It seems slightly odd to be on such a nice round number!

**Comment by Salamat-UK** [comment]
> Am not sure why #4054 was closed, referring to this issue? Have updated to the latest but still getting the Subscriptions issue! (On multiple devices)

Pleased to say, working again as usual, this morning. (9.00 GMT)  Many thanks!

**Comment by eberkopec** [comment]
It seems the issue is fixed. I got an update yesterday and everything is fine now.
    On Tuesday, November 5, 2024 at 05:13:42 AM EST, Salamat-UK ***@***.***> wrote:  
 
 



Am not sure why #4054 was closed, referring to this issue? Have updated to the latest but still getting the Subscriptions issue! (On multiple devices)


Pleased to say, working again as usual, this morning. (9.00 GMT) Many thanks!

**Comment by swakefield81** [comment]
While it was fixed for me this morning between around 8am and 10am GMT I've just tried using it now and its broken again

**Comment by Salamat-UK** [comment]
Same for me, on all devices

**Comment by dontwantonebutneedone** [comment]
I solved this problem by going to the regular youtube client and changing the playlists to "unlisted" from "private". That fix has stuck through several recent upgrades.

**Comment by Bunjopang** [comment]
I have the issue that any playlist that isn't my own is being treated as a singular video with no way to play anything but the first video and no wag to see the playlist

**Comment by koffeedude** [comment]
I am still having the same issue with Fire TV Cube. I  have updated to the latest version 24.71 beta since I last added to this issue and nothing has resolved it. The circle spins and spins and nothing works. As I stated before I deleted cache, restarted and check for Fire TV updates each day and its still happening.
